src/Framework/src/Microsoft.AspNetCore.App.Runtime.csproj

```diff
@@ -119,6 +119,11 @@ This package is an internal implementation of the .NET Core SDK and is not meant
   </PropertyGroup>
 
   <ItemGroup>
+    <!-- Workaround for https://github.com/dotnet/core-setup/issues/5665 -->
+    <ItemGroup>
+      <PackageConflictPlatformManifests Include="hackedManifest.txt" />
```

Comment:
File a bug on our end to fix this ... Using a hacked manifest makes me super nervous :).

Reply:
I want to see if this workaround even works. I couldn't test locally due to some machine setup issues but I wanted to unblock the build asap so I'm letting the CI test the fix for me.

Reply:
Looks like it worked so filed https://github.com/aspnet/AspNetCore-Internal/issues/2152